lib/LDKeyPair.js

```diff
@@ -89,7 +118,18 @@ class LDKeyPair {
         throw new Error(`Unsupported Key Type: ${options.type}`);
     }
   }
-
+  /**
+   * @description Generates a
+   * [pdkdf2]{@link https://en.wikipedia.org/wiki/PBKDF2} key.
+   * @param {string} password - The password for the key.
+   * @param {string} salt - Noise used to randomize the key.
+   * @param {number} iterations - The number of times to run the algorithm.
+   * @param {number} keySize - The bit length of the key.
```

Comment:
keySize is in bytes.

Reply:
are you sure about that? I think it might be both. I got that from the docs for one of the key generation functions. so I will double check this.

Reply:
you were 100% correct on this also added a link to the forge repo for this encryption type.